packages/okta-react/README.md

```diff
@@ -267,6 +266,55 @@ class App extends Component {
 export default App;
 ```
 
+#### PKCE Example
+
+Assuming you have configured your application to allow the `Authorization code` grant type, simply pass `pkce=true` to the `Security` component. This will configure the `Auth` object to perform PKCE flow for both login and token refresh.
+
+```typescript
```

Comment:
Why are these all flagged as TS instead JS?

Reply:
it makes the jsx tags prettier

Reply:
The dev blog supports jsx as a language type. Maybe the new docs site does too?

Reply:
looks like it is: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml